### PR TITLE
feature[docs]: Build documentation automatically

### PR DESCRIPTION
**Pull request by Adriana618**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+3. To confirm the CLI was installed properly, run the following CLI command:
+```bash
+$ estela --version
+```
+If the installation was successful, you should see the following output:
+```
+$ estela --version
+estela, version 0.1
```
Comment (mgonnav) [inline docs/installation.md:44]: Let's make the version text also be updated here when we use `bumpversion`

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+plugins:
+  - search
+
+nav:
+  - Home: index.md
+  - Installation: installation.md
+  - Quickstart: quickstart.md
+  - Configuration: configuration.md
+  - Command Line Interface: cli.md
+  - Return estela Docs: https://estela.bitmaker.la/docs
```
Comment (mgonnav) [inline mkdocs.yml:24]: Let's change this to "Return to estela Docs"

File: docs/installation.md (hunk 1 above)
Comment (Adriana618) [reply docs/installation.md]: Done

**Review by mgonnav — CHANGES_REQUESTED** [review]
The command docstrings are not being correctly displayed in the generated docs. For example, for the command `estela list cronjob`, the docstring is:
```
    """List jobs of a given spider
    \b
    SID is the spider's sid
    PID is the project's pid (active project by default)
    TAG is the tag used to filter jobs
    """
```
However, in the documentation it appears as:
```
List cronjobs of a given spider

SID is the spider's sid PID is the project's pid (active project by default) TAG is the tag used to filter cronjobs
```
Please fix this.

File: requirements/dev.txt [hunk 3]
```diff
@@ -22,13 +22,24 @@ click==8.1.0
...
     #   -r test.txt
     #   black
+    #   mkdocs
+    #   mkdocs-click
     #   pip-tools
+<<<<<<< HEAD
 docker==6.1.3
     # via -r test.txt
 idna==3.4
+=======
```
Comment (mgonnav) [inline requirements/dev.txt:32]: There seem to be conflicts in the requirements that have not been resolved. Please update this as well.

Comment (Adriana618) [reply requirements/dev.txt]: Done.

**Review by mgonnav — APPROVED** [review]
Well done!

**Pull request merged**